code/modules/recycling/sortingmachinery.dm

```diff
@@ -79,6 +79,9 @@
 	desc = "A brown paper delivery parcel."
 	icon = 'icons/obj/storage.dmi'
 	icon_state = "deliverypackage3"
+	item_state = "deliverypackage"
+	lefthand_file = 'icons/mob/inhands/items_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/items_righthand.dmi'

```

Comment:
Actually these are unnecessary, they're inherited from `/obj/item`
https://github.com/tgstation/tgstation/blob/c6281dcaef4c8c35bc9603e88b4efa57b86b3848/code/game/objects/items.dm#L7-L12

Reply:
Right, so I should just set their item state then, correct?